manage.py

```diff
@@ -154,7 +154,6 @@ def run_patches():
         GeventCommand('ptop_preindex'),
         GeventCommand('sync_prepare_couchdb_multi'),
         GeventCommand('sync_couch_views'),
-        GeventCommand('celery', contains='-P gevent'),
```

Comment:
Interesting. This is used below to decide whether to apply the gevent money patch, which makes gevent work. With the new way of running celery, how are we making sure to apply the monkey patch?

Reply:
celery actually calls the monkey patch on its own when the `-P gevent` flag is passed

Reply:
Oh, neat. So we possibly never actually needed this line / we were double-patching? (Which was probably fine I'd think.)

Reply:
I suspect it was necessary on django-celery, but not celery itself.